lib/plugins/features/movies/controller.js

```diff
@@ -27,3 +30,19 @@ exports.list = (query) => {
   })
   .fetchAll();
 };
+
+exports.addLocation = (payload) => {
+  return Bluebird.all([
+    new Location({ id: payload.location_id }).fetch(),
```

Comment:
We usually [omit the `_id`](https://github.com/lob/wiki/blob/master/style-guides/api.md#2-a-parameter-should-never-have-_id) from payload params.